### PR TITLE
Add topic about Data Ingestion to Learn

### LINKED ISSUE #181: Create "Ingest data" section under the "Learn" group
Currently, Console has a drawer that lists links to various data ingestion resources.

image

The links go to:
* https://github.com/vectara/vectara-ingest
* https://github.com/vectara/getting-started
* https://docs.vectara.com/docs/api-reference/indexing-apis/indexing
* https://docs.vectara.com/docs/api-reference/indexing-apis/file-upload/format-for-upload
* https://docs.vectara.com/docs/api-reference/indexing-apis/file-upload/file-upload

The intention here is to give the user a single place to learn about data ingestion, the various options for ingesting data, and to make a selection. If we could fulfill this purpose with a page in the docs then we can remove this drawer from Console and replace it with a link to this page.

### PR DESCRIPTION
**Pull request by pwoznic**
Added topic about data ingestion to replace info in Console drawer and fixes #181

### CODE REVIEW

File: www/docs/learn/data-ingestion.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+files into a corpus. We recommend this option when you do not need to define 
+additional user-supplied metadata beyond what is extracted by the Vectara 
+platform.
+
+Our [interactive API Playground](/docs/rest-api/file-upload) enables you
+to experiment with this File Upload REST endpoint.
+
+## Attach Metadata with JSON
+
+You can [format your data as JSON](/docs/api-reference/indexing-apis/file-upload/format-for-upload), which enables you to attach metadata for
```
Comment (cjcenizal) [inline www/docs/learn/data-ingestion.md:46]: This link is to a child page of the "File Upload API" page. Should this section be nested under the "File Upload API" section above? As an H2, it's presented as if it's an alternative to the API options above.

**Review by cjcenizal — CHANGES_REQUESTED** [review]
Thanks for adding this info Paul! I reviewed this page with the assumption that page's goal is to help the user decide which method to use, and so I left a couple comments on things that I had a hard time associating with that goal.

File: www/docs/learn/data-ingestion.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+efficiently index their data and leverage our advanced search capabilities.
+
+## Vectara Ingest: Sample Data Ingestion Framework
+
+Getting data into Vectara is simple using either our REST or gRPC APIs. We 
+built a full sample ingestion framework ready to go with [Vectara Ingest](https://github.com/vectara/vectara-ingest), which 
+includes preconfigured templates that enable you to pull data from many 
+popular data sources such as websites and RSS feeds.
+
+## Getting Started with Examples
```
[the remark this answers is not in the document]
Comment (pwoznic) [reply www/docs/learn/data-ingestion.md:20]: I've thought a bit about it, and I think I will remove the GS reference from this topic and keep it focused on data ingestion.

**Review by pwoznic — COMMENTED** [review]
Updated topic

File: www/docs/learn/data-ingestion.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+Getting data into Vectara is simple using either our REST or gRPC APIs. We 
+built a full sample ingestion framework ready to go with [Vectara Ingest](https://github.com/vectara/vectara-ingest), which 
+includes preconfigured templates that enable you to pull data from many 
+popular data sources such as websites and RSS feeds.
+
+## Standard Indexing API
+
+We recommend the [Standard Indexing](/docs/api-reference/indexing-apis/indexing) method for 
+indexing a set of related documents or content into a corpus.
```
Comment (cjcenizal) [inline www/docs/learn/data-ingestion.md:23]: I'm having trouble reasoning about this guidance. Does this imply the File Upload API is for indexing documents that aren't related? How can we provide more details that help folks choose one or the other?

For reference this is what we show in Console:

image

File: www/docs/learn/data-ingestion.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+You can also experiment with this REST endpoint in our interactive [API Playground](/docs/rest-api/index).
+
+## File Upload API
+
+The [File Upload](/docs/api-reference/indexing-apis/file-upload/file-upload) method exposes an 
+HTTP endpoint to upload and index files into a corpus. We recommend this 
+option when you do not need to define additional user-supplied metadata beyond 
+what is extracted by the Vectara platform.
```
Comment (cjcenizal) [inline www/docs/learn/data-ingestion.md:32]: It sounds like this is best for situations where you're not manually defining metadata. But how will the reader square that with the "Attach Metadata with JSON" information? It sounds contradictory or at least a bit confusing.

**Review by cjcenizal — APPROVED** [review]
Thanks for making those changes. I think this is a good step forward. I left comments on a couple other points that confused me but I don't consider them blockers.

**Pull request merged**